requirements/static/ci/windows.in

```diff
@@ -5,3 +5,6 @@ pygit2<=0.28.2; python_version < '3.8'
 pygit2<1.1.0; python_version == '3.8'
 pygit2>=1.2.0; python_version >= '3.9'
 sed
+bcrypt==3.2.0
+passlib==1.7.4
```

Comment:
Shouldn't these become windows packaging requirements too?
/cc @twangboy

Reply:
Potentially. I added them since the tests were changed and made use of these packages, whereas previously Window's did not need them. Ill probably take this off the AL plate as not make a hasty choice

Reply:
This still seems to be unresolved. Do we need to add them to the packages?

Reply:
Let me remove them and run the tests once more. They were added to support previous changes in the test file that may not be needed any longer.